### DISCUSSION TITLE
Energy Reading for Delta 2

### DISCUSSION THREAD

**Discussion by viv-01** [tolwi/hassio-ecoflow-cloud#91] [General]
Hi

Just wondering if there are enery reading for the Delta 2. I tried adding it to the energy dashboard but it doesn't pickup?
![Screenshot 2023-07-20 145117](https://github.com/tolwi/hassio-ecoflow-cloud/assets/63659554/0a7ab800-3bb7-4586-ba9a-14d4facdbb3c)

**Comment by tolwi** [comment]
you should use [Integration - Riemann sum integral](https://www.home-assistant.io/integrations/integration/?#energy)
